ci/unit_tests.sh

```diff
@@ -1,18 +1,13 @@
 #!/bin/bash
 
-source ./ci/helpers.sh
-
 PROJECT_NAME="insights-chrome"
 TEST_CONT="${PROJECT_NAME}-unit-tests"
 IMG_TAG=$(git rev-parse --short=8 HEAD)
 
-docker build -t "${TEST_CONT}:${IMG_TAG}" -f Dockerfile.test .
+docker run --name "${TEST_CONT}-${IMG_TAG}" -d -i "${NODE_BASE_IMAGE}" /bin/sh
+
+docker cp . "${TEST_CONT}-${IMG_TAG}":. 
```

Comment:
when copying files with `cp` you have to specify an absolute path, hence now they are copying into the container's root, where the user doesn't have permissions. Adding the prefix `/opt/app-root/src/` should do the trick.

the same goes for the `exec` command, you can specify where the command will be run with the `-w` flag.